code/game/gamemodes/meteor/meteor.dm

```diff
@@ -69,6 +69,7 @@
 		universe.OnExit()
 
 	var/datum/universal_state/meteor_storm/meteor_storm = new /datum/universal_state/meteor_storm
+	universe = meteor_storm
```

Comment:
Alternatively, you could pass an existing datum to the proc instead of a typepath and add behaviour for that.